base.yaml

```diff
@@ -110,7 +83,7 @@ cp_zookeeper:
 cp_kafka:
   _install: true
   persistence:
-    size: 3Gi
+    size: 30Gi
```

Comment:
Do we need this much storage for a basic installation?

Reply:
I can decrease it to 10, but 3 will be too little quite quickly.